authorized_keys_command.sh

```diff
@@ -4,10 +4,13 @@ if [ -z "$1" ]; then
   exit 1
 fi
 
+# source configuration if it exists
+[ -f /etc/sysconfig/aws-ec2-ssh ] && . /etc/sysconfig/aws-ec2-ssh
```

Comment:
I am not 100% sure if we should use this location, or just a file in `/etc/`. Now I think of it some more, the sysconfig directory is very centos/redhat specific.

What do you think?